coalib/processes/Processing.py

```diff
@@ -344,7 +347,35 @@ def instantiate_processes(section,
         log_printer,
         ignored_file_paths=glob_list(section.get('ignore', "")),
         limit_file_paths=glob_list(section.get('limit_files', "")))
+
+    # This stores all matched files irrespective of whether coala is run
+    # only on changed files or not. Global bears require all the files
+    complete_filename_list = filename_list
+
+    # Start tracking all the files
+    if cache:
```

Comment:
No, it is wrapped around a if...else clause that makes sure cache isn't `None` when passed.

And `cache` becomes `None` only in irrelevant tests except for once in DbusDocument: I remember asking @AbdealiJK and I think he said it should be fine IIRC (I'm not caught up on Dbus though).